File: java/app/src/main/java/io/baff/sample/SubActivity.java
package io.baff.sample;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import io.baff.core.GetBaff;

public class SubActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sub);
    }

    public void onRunGetBaff(View view) {
        GetBaff.ShowGetBaff(this);
    }
}

File: kotlin/app/src/main/java/io/baff/sample/SubActivity.kt
package io.baff.sample

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import io.baff.core.GetBaff

class SubActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_sub)
    }

    fun onRunGetBaff(view: View?) {
        GetBaff.ShowGetBaff(this)
    }
}